### DISCUSSION TITLE
electron-vite的增量更新应该如何实现

### DISCUSSION THREAD

**Discussion by ghost** [alex8088/electron-vite#239] [Q&A]
electron-vite的增量更新应该如何实现

**Comment by alex8088** [comment]
是否指 Electron 程序更新？

这实际和 electron-vite 构建工具无关。但它可以辅助你实现这个想法。增量更新程序文件，意味着你需要将经常要更新的业务代码，和不需要经常更新的代码，拆分打包，这一部分你可以参考 
https://cn.electron-vite.org/guide/build.html#%E5%88%86%E5%9D%97%E7%AD%96%E7%95%A5
增量更新业务代码即可，如何拆分，可由自己决定。
但实现增量更新比如下载升级文件，如何替换旧文件等，这些都是要自己实现的，实际也远不止这些问题要解决，比如如何解决文件缓存问题等等。没有现成的完备方案。

但做一个桌面端软件，我觉得对增量这个意义并不大，如果还需要考虑大小和流量消耗，那用web更适合。

建议还是使用 electron-builder，它提供完善的打包和升级体系。
